torchaudio/io/_compat.py

```diff
@@ -2,7 +2,61 @@
 
 import torch
 import torchaudio
-from torchaudio.backend.common import AudioMetaData
+
+
+class AudioMetaData:
+    """Return type of ``torchaudio.info`` function.
+
+    This class is used by :py:mod:`"sox_io" backend<torchaudio.backends.sox_io_backend>` and
+    :py:mod:`"soundfile" backend<torchaudio.backends.soundfile_backend>`.
+
+    :ivar int sample_rate: Sample rate
+    :ivar int num_frames: The number of frames
+    :ivar int num_channels: The number of channels
+    :ivar int bits_per_sample: The number of bits per sample. This is 0 for lossy formats,
+        or when it cannot be accurately inferred.
+    :ivar str encoding: Audio encoding
+        The values encoding can take are one of the following:
+
+            * ``PCM_S``: Signed integer linear PCM
+            * ``PCM_U``: Unsigned integer linear PCM
+            * ``PCM_F``: Floating point linear PCM
+            * ``FLAC``: Flac, Free Lossless Audio Codec
+            * ``ULAW``: Mu-law
+            * ``ALAW``: A-law
+            * ``MP3`` : MP3, MPEG-1 Audio Layer III
+            * ``VORBIS``: OGG Vorbis
+            * ``AMR_WB``: Adaptive Multi-Rate
+            * ``AMR_NB``: Adaptive Multi-Rate Wideband
+            * ``OPUS``: Opus
+            * ``HTK``: Single channel 16-bit PCM
+            * ``UNKNOWN`` : None of above
```

Comment:
Later we need to test that these formats are same regardless of the backend picked implicitly, at least for common ones.